dspace/modules/xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/submit/OverviewStep.java

```diff
@@ -62,19 +63,24 @@
     public void addBody(Body body) throws SAXException, WingException, UIException, SQLException, IOException, AuthorizeException {
         Collection collection = submission.getCollection();
         String actionURL = contextPath + "/handle/" + collection.getHandle() + "/submit/" + knot.getId() + ".continue";
+        org.dspace.content.Item publication = DryadWorkflowUtils.getDataPackage(context, submission.getItem());
 
         body.addDivision("step-link","step-link").addPara(T_TRAIL);
+
         Division helpDivision = body.addDivision("general-help","general-help");
         helpDivision.setHead(T_MAIN_HEAD);
         helpDivision.addPara(T_MAIN_HELP);
+        if (publication.checkForDuplicateItems(context)) {
```

Comment:
I got a null pointer exception on this line. After submitting a duplicate item (same pub DOI), as a curator I claimed the task and attempted to edit the item metadata.